src/main/java/com/fauna/types/NamedDocumentRef.java

```diff
@@ -3,35 +3,45 @@
 import java.util.Objects;
 
 /**
- * Represents a document ref that has a "name" instead of an "id". For example, a Role document
- * reference is represented as a NamedRef.
+ * Represents a reference to a document identified by a "name" instead of an "id".
+ * This class is used for document references where a unique name
+ * (e.g., for a Role) is used instead of a numeric or autogenerated ID.
```

Comment:
```suggestion
 * This class is used for references to system collection documents where a unique name
 * (e.g., for a Role) is used instead of a numeric or autogenerated ID.
```